spyder-custom-layout/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+<!--- **PLEASE READ:** When submitting here, please ensure you've completed the following checklist and checked the boxes to confirm. Issue reports without it may be closed. Thanks! --->
+
+### Issue Report Checklist
+
+* [ ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
```

Comment:
Please remove this line because third-party plugin errors shouldn't be reported to Spyder.